lib/native-interaction/rubix-local.dart

```diff
@@ -367,16 +381,24 @@ class RubixLocal {
     RubixLog().appendLog("getTransactionByCount response: ${response.body}");
     List<dynamic> dataResponse = getRubixResponseJson(response);
 
-    Iterable<TxnSummary> txnSummaries = dataResponse.map((e) => TxnSummary(
-      txnId: e['txn'],
-      sender: e['senderDID'],
-      receiver: e['receiverDID'],
-      amount: e['amount'] is int ? e['amount'].toDouble() : e['amount'],
-      comment: e['comment'],
-    )).toList();
+    Iterable<TxnSummary> txnSummaries = dataResponse
+        .map((e) => TxnSummary(
+              txnId: e['txn'],
+              sender: e['senderDID'],
+              receiver: e['receiverDID'],
+              amount: e['amount'] is int ? e['amount'].toDouble() : e['amount'],
+              comment: e['comment'],
+            ))
+        .toList();
 
     return GetTransactionLogRes(
       txnLog: txnSummaries,
     );
   }
+
+  Future<File> base64ToImage(String base64Image, String path) async {
+    File file = File(path);
+    await file.writeAsBytes(base64Decode(base64Image));
```

Comment:
shall return without await here. call to base64ToImage shall be awaited, which is already being done - that alone is enough.